### PR TITLE
test-configs.yaml: Enable LTP-timers tests on devices in lab-collabora 

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

File: test-configs.yaml [hunk 1]
```diff
@@ -1592,6 +1603,7 @@ test_configs:
       - baseline
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:1606]: https://staging.kernelci.org/test/plan/id/5fd6ef51f728e28232992461/

File: test-configs.yaml [hunk 2]
```diff
@@ -1610,6 +1622,7 @@ test_configs:
       - baseline-nfs
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:1625]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6f56e30ae95308d992447/

File: test-configs.yaml [hunk 3]
```diff
@@ -1681,6 +1694,7 @@ test_configs:
       - baseline-nfs
       - kselftest
       - ltp-crypto
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:1697]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6ef96dbcf9ddd9299245c/

File: test-configs.yaml [hunk 4]
```diff
@@ -1744,6 +1758,7 @@ test_configs:
       - baseline-nfs
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:1761]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd756f83e8945e687992447/

File: test-configs.yaml [hunk 5]
```diff
@@ -1801,6 +1816,7 @@ test_configs:
...
```
Comment (Lakshmipathi) [inline test-configs.yaml:1819]: Tested OK on staging:   https://staging.kernelci.org/test/plan/id/5fd6ee5c5c80c7e5e5992477/

File: test-configs.yaml [hunk 6]
```diff
@@ -2069,6 +2086,7 @@ test_configs:
       - kselftest
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:2089]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6ed34e9b39cc4e199244d/

File: test-configs.yaml [hunk 7]
```diff
@@ -2087,6 +2105,7 @@ test_configs:
       - baseline-nfs
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:2108]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6ee6f66f20cf22699244d/

File: test-configs.yaml [hunk 8]
```diff
@@ -2104,6 +2123,7 @@ test_configs:
       - kselftest
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:2126]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6eeeacc9b12511f992453/

File: test-configs.yaml [hunk 9]
```diff
@@ -2157,6 +2177,7 @@ test_configs:
       - baseline
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:2180]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd6eccaf8e4af8d7a9925bf/

File: test-configs.yaml [hunk 10]
```diff
@@ -1930,6 +1946,7 @@ test_configs:
       - baseline-nfs
       - igt-kms-exynos
       - ltp-ipc
+      - ltp-timers
```
Comment (Lakshmipathi) [inline test-configs.yaml:1949]: Tested OK on staging:  https://staging.kernelci.org/test/plan/id/5fd83da6395c4eb98f99248e/

**Comment by Lakshmipathi** [comment]
Tested OK on staging: https://staging.kernelci.org/test/job/mainline/branch/master/kernel/v5.10-3208-g148842c98a24/plan/ltp-timers/

File: test-configs.yaml (hunk 1 above)
Comment (gctucker) [reply test-configs.yaml:1606]: Well, no that didn't work: https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20201219.0/arm/multi_v7_defconfig/gcc-8/lab-collabora/ltp-timers-bcm2836-rpi-2-b.html#L708

The board booted OK, but the `timers` tests failed to start.  It looks like something is missing in the rootfs.

File: test-configs.yaml (hunk 2 above)
Comment (gctucker) [reply test-configs.yaml:1625]: Same thing here, only `login` test case but nothing from LTP timers.

**Review by gctucker — COMMENTED** [review]
The jobs appear to be submitted correctly and they boot, but the tests aren't run.  Please check nothing is missing in the LTP rootfs and if so, create another PR to fix that.

**Comment by gctucker** [comment]
Skipping to get the `ltp-mm` PR merged first (#569).

**Pull request closed without merging**